code/modules/clothing/suits/armor_vr.dm

```diff
@@ -33,6 +33,48 @@
 				H << "<span class='warning'>You need to have a wolf-taur half to wear this.</span>"
 				return 0
 
+	kate
```

Comment:
I'd rather this be consistent. I'd also prefer this be correctly added to the fluff item files with the appropriate comments attached.